Xamarin.Forms.Platform.Android/AppCompat/NavigationPageRenderer.cs

```diff
@@ -468,7 +468,7 @@ void CurrentOnPropertyChanged(object sender, PropertyChangedEventArgs e)
 				ResetToolbar();
 		}
 
-		void HandleToolbarItemPropertyChanged(object sender, PropertyChangedEventArgs e)
+		protected virtual void HandleToolbarItemPropertyChanged(object sender, PropertyChangedEventArgs e)
```

Comment:
This doesn't need to be `protected virtual` for the user to customize the behavior of `UpdateDrawableMenuItem`; this should be private.

Reply:
I need this because if i create my own ToolBarItem with a new property, i need recreate menu.

```
public class CartCountToolBarItem : ToolBarItem
{
static readonly BindableProperty CartCountProperty = .....
}
```

i need call UpdateMenu when this property change for display cart count inside menu item.

```
protected override void HandleToolbarItemPropertyChanged(object sender, PropertyChangedEventArgs e)
		{
base.HandleToolbarItemPropertyChanged(sender, e);

			if (e.PropertyName == CartCountProperty.PropertyName)
				UpdateMenu();
		}
```

Reply:
To remain consistent this should be named OnToolbarItemPropertyChanged as our protected virtuals should be.

I am not a huge fan of propagating INPC further but its the hammer we have.

Reply:
@jassmith So are you cool with this being `protected virtual`? If you're cool with it, then I'm cool with it :)